### PR TITLE
Makefile targets to improve developer flow

### LINKED ISSUE #211: Common Commands/Testing Flows Put into Makefile
**Is your feature request related to a problem? Please describe.**
We should be using the `Makefile` to run commands that have to do with setting up specific pieces of the infra rather than just providing documentation around these things

- Some good examples of this are for running things like tilt, installing the chart, setting up the E2E testing environment, etc.

### PR DESCRIPTION
**Pull request by nitishm**
Closes #211 

Signed-off-by: Nitish Malhotra <nitish.malhotra@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,6 +14,16 @@ endif
 
 all: manager
 
+dev-up: dev-cluster dev-deploy
+dev-down: stop
+	kind delete cluster orkestra
```
Comment (jonathan-innis) [inline Makefile:19]: Does this command work? I thought you needed to specify `--name`?

**Review by jonathan-innis — CHANGES_REQUESTED** [review]
There's a command here that doesn't work, other than that looks good :)

File: Makefile (hunk 1 above)
Comment (nitishm) [reply Makefile:19]: yes - I need to push a new commit to this PR with the fixes and some better targets but I am hitting some snags. Will update this PR soon

**Comment by nitishm** [comment]
@jonathan-innis I made these changes but I do hit issues due to failing targets. I will this PR in WIP state until we clean it up a little more

**Pull request closed without merging**